src/client/datascience/jupyter/kernelVariables.ts

```diff
@@ -421,6 +422,37 @@ export class KernelVariables implements IJupyterVariables {
         return result;
     }
 
+    public async getVariableProperties(
+        word: string,
+        notebook: INotebook,
+        cancelToken: CancellationToken | undefined
+    ): Promise<JSONObject> {
+        const matchingVariable = await this.getMatchingVariable(word, notebook, cancelToken);
+        const settings = this.configService.getSettings(notebook.resource).variableTooltipFields;
+        const languageSettings = settings['python'] as JSONObject;
```

Comment:
Hardcoding python here shouldn't be necessary. Other code in this file tries to be language agnostic.

Reply:
Oh wait, didn't see this. GitHub shows outdated code snippets for stuff that's been reviewed, so I think you were looking at the old version. I initially passed the language through from the hover provider request, but I much prefer this idea and it's more consistent with what the rest of kernelVariables does. Thanks for the suggestion :)